weasel/tests/test_cli/test_base.py

```diff
@@ -0,0 +1,87 @@
+import pytest
```

Comment:
Again, it would be helpful to know which of these tests have been copied from spaCy and simply distributed among several Python files and which are completely new.